### ISSUE TITLE
Angular 9/Ivy warnings

### ISSUE THREAD

**Issue by ThomasDevens** [dhilt/ngx-ui-scroll#148]
Getting this warning when running angular's postinstall script `ngcc --properties es2015 browser module main --first-only --create-ivy-entry-points`:

```
Warning: Entry point 'ngx-ui-scroll' contains deep imports into '{PATH_TO_PROJECT}/node_modules/rxjs/index'. This is probably not a problem, but may cause the compilation of entry points to be out of order.
```

**Comment by dhilt** [comment]
@ThomasDevens Hi, I just tried run ngx-ui-scroll v1.4.2 over new Angular 9 application and did not find any issues, how can I reproduce your particular case? I guess it might be something important, so I would appreciate if yo can provide more details.

**Comment by dhilt** [comment]
@ThomasDevens Thanks for the report! I fixed deep-imports issue and released ngx-ui-scroll v1.4.3 containing the fix. Would be great if you could confirm the issue is gone.

**Comment by ThomasDevens** [comment]
Sorry for the delayed response; Confirmed warnings are gone, thanks for the quick response!
